AI/VCAI/Pathfinding/AINodeStorage.h

```diff
@@ -62,6 +62,7 @@ class AINodeStorage : public INodeStorage
 	/// 1-3 - position on map, 4 - layer (air, water, land), 5 - chain (normal, battle, spellcast and combinations)
 	boost::multi_array<AIPathNode, 5> nodes;
 	const CPlayerSpecificInfoCallback * cb;
+	VCAI * ai;
```

Comment:
Why is it not const? Non const `CGameInterface` instances should be used only for sending request to server

Reply:
Ok will check

Reply:
Fixed